### ISSUE TITLE
Changing character name also resets the map

### ISSUE THREAD

**Issue by Afura33** [WalterCouto/D2CE#4]
Hello, I am playing diablo 2 resurrected on pc and I love the editor so far, thanks. I did encounter a bug, everytime I change the name of a character it also resets the map so I have to reveal the map again, the weird thing is that it is still the same map as before, but I have to reveal all again. I am on windows 10 pro and I am using the latest version of diablo 2 resurrected and the latest verison of the editor (Version 2.16.). 

Also I do have a question, if I change the character name it creates two files with the old character name. One of the files is a BAK file with the old character name that has a long number behind it and the second file is a ctl file with the old character name. I did delete both files and nothing bad happened, but I am not sure if this is a good idea, can I delete these 2 files? 

Thanks for your work, I appreciate it.

**Comment by WalterCouto** [comment]
The back file is always safe to delete, it is there in case a change corrupted the character file. If that happened, you could just delete the character file and rename the back file.

That ctl file may be the reason for this bug, that should have renamed as well. There are a few extensions attached to the character file and for a rename they all need to be renamed. If you try this rename again, you can try to manual rename the ctl file and see if that resolves the issue.

I'll try to fix this issue in the code as well.

**Comment by Afura33** [comment]
Hello Walter,

Sorry for my late reply. Ok this is good to know, I didn't know I could get my character back this way, thank you

Hmmm yea there is a ctl file of the old character name and a ctl file of the new character name. I tried to rename the old ctl file with the new name, but this didn't work, it says that this ctl file with this name already exist. I added a screenshot to see how it looks like in my save games: https://ibb.co/GHq3nY4 
_Zookeeper_ was the old name and _Xero_ is the new name.
